docs/gateway.md

```diff
@@ -80,5 +110,5 @@ here the backend is the service `el-pipelines-as-code-interceptor` in the `opens
 ## Limitations
 
 - There is currently no controller watching EventListeners to configure the gateway dynamically. This means that the gateway would work for Pipelines as Code, which offers a stable entrypoint but not for pure Tekton Triggers.
-- PipelineRuns are not visible in any kcp workspace.
+- PipelineRuns created by Pipeline As Code are not visible in any kcp workspace.
```

Comment:
[Status of PipelineRuns](https://pipelinesascode.com/docs/guide/statuses/) are reported back to the git repository. Example: https://github.com/openshift-pipelines/pipelines-as-code/pull/784/checks?check_run_id=8152559456
Technically users may have connectivity to the compute clusters but may miss name resolution and credentials to authenticate against the console and visualize the details.

Reply:
Here's a question - when users configure the `Repository` CRD, will they do it in KCP or directly on the workload cluster?

Reply:
Ideally they should not know about the workload cluster. They should configure the Repository CRD in kcp. I have added it to the list of resources to sync. This will then get picked up by the PaC controller. The PipelineRun executes in the namespace where the Repository resource was synced.

Reply:
I still feel that it is a UX issue that only the summary / result gets reported to GitHub.
This is however rather a PaC issue than a Pipeline-Service one. Independent of our integration with kcp it is quite a big bet to assume that every user possibly creating a PR to a public repository have credentials to access the CI infrastructure used by the repo.